addOns/automation/src/main/java/org/zaproxy/addon/automation/jobs/RequestorJob.java

```diff
@@ -120,14 +120,17 @@ public void verifyParameters(AutomationProgress progress) {
                                     req.getResponseCode(),
                                     request));
                 }
-                this.getData().addRequest(req);
+                this.getData().addRequest(req);/** The req object contains the Request object and its being passed to the requests arraylist which is being maintained in this class */
```

Comment:
You can rename `req` ➡️ `request` which addresses the first half of the comment.
Is the second part really that much of a leap based on `getData().addRequest(request)`? To me it seems evident request is being added to the job's data... but maybe I'm over simplifying in my head.

Reply:
`request` variable is already there, so changing `req` to `request` is gonna require some work.

Reply:
Well it could be `modifiedRequest` or `newRequest` I dunno, I'd have to read through again but there should be a reasonable way to make it more obvious.

Reply:
Agreed!